op-mode-definitions/show-system.xml.in

```diff
@@ -66,6 +66,55 @@
               <help>Show user accounts</help>
             </properties>
             <children>
+              <node name="authentication">
+                <properties>
+                  <help>Show user account authentication information</help>
+                </properties>
+                <children>
+                  <tagNode name="user">
+                    <properties>
+                      <help>Show configured user</help>
+                      <completionHelp>
+                        <script>${vyos_libexec_dir}/vyos-sudo.py ${vyos_op_scripts_dir}/show_users.py | awk '{print $1}' | tail -n +3</script>
```

Comment:
Can we use `path` ?
```
<path>system login user</path>
```

Reply:
Thank you!
It's really much better that way :)
Fixed